charts/odpi-egeria-lab/values.yaml

```diff
@@ -31,7 +31,7 @@ service:
 egeria:
   #logging: OFF
   development: true
-  version: "3.13"
+  version: "3.14"
```

Comment:
See 'gitTagForNotebooks' --
a) Need to create a new tag in git for 314
b) need to update the lab chart to use this tag

For testing you can just specify the tag as 'main'

Reply:
There is also a jupyter image we build, and specify - but it may be the current version is the latest

Reply:
> See 'gitTagForNotebooks' -- a) Need to create a new tag in git for 314 b) need to update the lab chart to use this tag
> 
> For testing you can just specify the tag as 'main'

The current main is not stable, tested and ui-lab fails needs fixing first. this is pre-release version for testing

Reply:
The minor problem is now fixed https://github.com/odpi/egeria-jupyter-notebooks/pull/35.
About gitTagForNotebooks, we need to complete the test first. Shall I use `main` for now until we are sure and complete the test notebooks step for the release?

Reply:
Yes You could use main, or could leave it at old version and override on deployment - I think either is fine as it's a prerelease.

I just wanted to point it out, as it's something I added recently

Reply:
> There is also a jupyter image we build, and specify - but it may be the current version is the latest
the current version is latest